_episodes_rmd/05-day4.Rmd

```diff
@@ -1,135 +1,173 @@
 ---
 title: 'Day 4: Exploring Vector Data'
+exercises: 7
+questions: 
+- "Are two datasets from two different sources suitable to be plotted together?" 
+- "What spatial patterns can we see by visual analysis?"  
+- "What are some plotting options to impact the way data are visualized?" 
 output: html_document
 objectives:
-- "Importing vector data; exploring vector data through attributes and by plotting"
+- "Importing vector data and visually explore the vector data through attributes and by visualization."
+keypoints: 
+ - "exploring vector data attributes"
+ - "consider design options for your vector data"
+ - "visual analysis of plotted data"
```

Comment:
For these key points, it may be helpful to phrase them as meaningful things for them to remember. I adapted and selected some of the key points from the lessons. It was helpful to use what existed so I didn't feel like I had to make up something new. Here's a lesson that may be relevant for some key points (at the bottom): https://uw-madison-datascience.github.io/r-raster-vector-geospatial/06-vector-open-shapefile-in-r/index.html